source/content/wordpress-cron.md

```diff
@@ -71,62 +86,79 @@ terminus wp <site>.<env> -- cron <your wp-cron command and switches here>
+You might have the ability to set up multiple jobs at different times depending on the service you use. Creating a single job that calls your site's `wp-cron.php` script every 15 minutes is all you should need. WP-Cron will take care of the rest. You will need to adjust your job if you create new schedules that need to run more frequently than every 15 minutes.
 
-It's important that you do not add a value to the `doing_wp_cron` query variable; it must be empty for the cron to work properly.
+<Alert title="Note" type="info">
+
+Do not add a value to the `doing_wp_cron` query variable. This variable must be empty for Cron to work correctly.
+
+</Alert>
 
 ### Using Your Own Server
 
-If you administer your own server, you can use the cron service to make a call to the `wp-cron.php` script. You will have to learn how to properly set up a Cron job and use something like `wget` or `curl` to fetch a web page. Unless you take special precautions, it is not any safer to use your own server vs. a web-based cron service; however, it does give you more control.
+ You can use the Cron service to make a call to the `wp-cron.php` script if you administer your own server. You will have to learn how to correctly set up a Cron job and use something like `wget` or `curl` to fetch a web page. Using your own server is not safer than using a web-based Cron service.
+ 
+## Troubleshooting 
 
-### Security
+### Problems With Low Traffic Sites
 
-For the most part, `wp-cron.php` is secure. There is only one parameter you can pass in that will affect the script (`doing_wp_cron`). This `$_GET` value is not filtered; it is only used as a flag and not as input for a process or variable. Beyond that, all input is ignored.
+Low traffic WordPress sites may experience skipped jobs because WP-Cron can't execute jobs if people are not visiting your site. This doesn't mean your page will be slow from previous jobs when someone eventually does visit your site. Regardless of how many jobs WP-Cron has to execute, all jobs are run in the background so your site's performance is not adversely affected.
+
+Low traffic WordPress sites using Pantheon Cron may also experience skipped jobs. For example, a site that receives no traffic for twelve hours might be put to sleep, which will stop Pantheon Cron jobs. Pantheon Cron will continue to run every hour regardless of traffic for twelve hours.
+
+### Problems With High Traffic Sites
 
-While no script on a server attached to the Internet is 100% secure, `wp-cron.php` currently does not have any known vulnerabilities or exploits.
+ You might run into problems with WP-Cron if your WordPress-powered site is high traffic. The main issues that come up are **race conditions** and **long running processes**.
```

Comment:
This section looks okay to me.  I'd add a note to see the section on `Disable WP-Cron` to limit the occurrences of these issues.